src/applications/vaos/tests/appointment-list/components/AppointmentsPage/AppointmentsPageV2.unit.spec.jsx

```diff
@@ -437,4 +442,154 @@ describe('VAOS <AppointmentsPageV2>', () => {
     ).to.be.ok;
     expect(screen.getByRole('link', { name: 'Request appointment' }));
   });
+
+  it('should navigate to confirmed appointments detail page', async () => {
+    // VA appointment id from confirmed_va.json
+    const url = '/va/var21cdc6741c00ac67b6cbf6b972d084c1';
+
+    const appointment = getVAAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      startDate: moment().format(),
+      clinicId: '308',
+      clinicFriendlyName: "Jennie's Lab",
+      facilityId: '983',
+      sta6aid: '983GC',
+      communityCare: false,
+      vdsAppointments: [
+        {
+          bookingNote: 'New issue: ASAP',
+          appointmentLength: '60',
+          appointmentTime: '2021-12-07T16:00:00Z',
+          clinic: {
+            name: 'CHY OPT VAR1',
+            askForCheckIn: false,
+            facilityCode: '983',
+          },
+          type: 'REGULAR',
+          currentStatus: 'NO ACTION TAKEN/TODAY',
+        },
+      ],
+      vvsAppointments: [],
+    };
+
+    mockAppointmentInfo({
+      va: [appointment],
+      cc: [],
+      requests: [],
+      isHomepageRefresh: true,
+    });
+
+    const facility = {
+      id: 'vha_442GC',
+      attributes: {
+        ...getVAFacilityMock().attributes,
+        type: 'facility',
+        address: {
+          mailing: {},
+          physical: {
+            zip: '80526-8108',
+            city: 'Fort Collins',
+            state: 'CO',
+            address1: '2509 Research Boulevard',
+            address2: null,
+            address3: null,
+          },
+        },
+        id: 'vha_442GC',
+        name: 'Fort Collins VA Clinic',
+        phone: {
```

Comment:
I think you could leave the phone numbers you're not using out of the data here, to shorten the test up a little

Reply:
Done.